src/main/charts/bitbucket/README.md

```diff
@@ -1,6 +1,6 @@
 # bitbucket
 
-![Version: 0.13.0](https://img.shields.io/badge/Version-0.13.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 7.12.1-jdk11](https://img.shields.io/badge/AppVersion-7.12.1--jdk11-informational?style=flat-square)
+![Version: 0.13.0](https://img.shields.io/badge/Version-0.13.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 7.14.1-jdk11](https://img.shields.io/badge/AppVersion-7.14.1--jdk11-informational?style=flat-square)
```

Comment:
Are we updating the 0.13.0 version?

Reply:
There will be a new release created after this. I can see what you mean, the README.md is not updated until there is a new version released and then it will sow `0.14.0`. I don't want to couple this PR with solving that problem as it was happening before as well and I am planning to run the release today but it is something we should consider.

Reply:
Ahhh I see now, yep that makes sense 👍